src/backend/weaviate/weaviate_manager.py

```diff
@@ -11,8 +11,9 @@ class WeaviateManager:
     def __init__(self):
 
         try:
+            weaviate_port = os.getenv('WEAVIATE_PORT', '8080')  # Default to 8080 if not set
             self.client = Client(
-                os.getenv('WEAVIATE_HOST', 'http://weaviate:8080')
```

Comment:
You dont need a weaviate_host and only get the weaviate port without the default value

Reply:
I've modified all the changes as suggested. Removed the unnecessary Weaviate host and updated the Weaviate port to not include a default value.